try:
    print(number)
except:
    print("An error occurred")


num1 = 39
num2 = 13
try:
    print(num1 / num2)
except:
    print("Zero Division error has occurred")
finally:
    print("Success")




